### PR TITLE
dev-python/btrfsutil: new package, add 6.9

### PR DESCRIPTION
**Pull request by floppym**
Closes: https://bugs.gentoo.org/934567

### CODE REVIEW

File: dev-python/btrfsutil/btrfsutil-6.9.ebuild [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~ia64 ~loong ~mips ~ppc ~ppc64 ~riscv ~sparc ~x86"
+
+DEPEND="sys-fs/btrfs-progs"
+RDEPEND="${DEPEND}"
+
+distutils_enable_tests unittest
+
+src_unpack() {
+	if in_iuse verify-sig && use verify-sig ; then
```
Comment (mgorny) [inline dev-python/btrfsutil/btrfsutil-6.9.ebuild:35]: Err, why do you need to check `in_iuse`? oO

File: dev-python/btrfsutil/btrfsutil-6.9.ebuild [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+		# Upstream sign the decompressed .tar
+		# Let's do it separately in ${T} then cleanup to avoid external
+		# effects on normal unpack.
+		cp "${DISTDIR}"/${MY_P}.tar.xz . || die
+		xz -d ${MY_P}.tar.xz || die
+		verify-sig_verify_detached ${MY_P}.tar "${DISTDIR}"/${MY_P}.tar.sign
+
+		popd &>/dev/null || die
+		rm -r "${T}"/verify-sig || die
```
Comment (mgorny) [inline dev-python/btrfsutil/btrfsutil-6.9.ebuild:47]: To be honest, this approach seems wasteful. Why not just unpack the decompressed source here?

File: dev-python/btrfsutil/btrfsutil-6.9.ebuild (hunk 1 above)
Comment (floppym) [reply dev-python/btrfsutil/btrfsutil-6.9.ebuild:35]: src_unpack was copy/pasted from sys-fs/btrfs-progs. The in_iuse call is there to account for the live ebuild, which has no signature to verify.

Comment (floppym) [reply dev-python/btrfsutil/btrfsutil-6.9.ebuild:35]: (I will simplify this)

Comment (eli-schwartz) [inline dev-python/btrfsutil/btrfsutil-6.9.ebuild:35]: Copy-paste from btrfs-progs which conditionally inherits verify-sig to support being a live ebuild.

File: dev-python/btrfsutil/btrfsutil-6.9.ebuild (hunk 2 above)
Comment (floppym) [reply dev-python/btrfsutil/btrfsutil-6.9.ebuild:48]: Done.

Comment (mgorny) [reply dev-python/btrfsutil/btrfsutil-6.9.ebuild:48]: Actually, I think `sys-kernel/vanilla-kernel` tackles the same problem with a single streaming call.

Comment (thesamesam) [reply dev-python/btrfsutil/btrfsutil-6.9.ebuild:48]: Thanks, I've done that now. Nice suggestion.

**Pull request closed without merging**